__tests__/app.test.js

```diff
@@ -33,7 +33,42 @@ describe("API endpoint", () => {
       .get("/api")
       .expect(200)
       .then(({ body }) => {
-        expect(body.endpoints).toEqual(endpoints)
+        expect(body.endpoints).toEqual(endpoints);
+      });
+  });
+});
+
+describe("Articles endpoint", () => {
+  test("GET:200 - Returns the provided article", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        body.article.forEach((result) => {
```

Comment:
Because this is a specific article, and we have access to our test data and know it will be the same every time the tests are run - we can afford to be more specific with what we're expecting back from this endpoint